src/main/scala/ClapYourHands.scala

```diff
@@ -1,2 +1,130 @@
-object ClapYourHands extends App {
+//All the scala newbie things
+object scalaNewbieProjects extends App{
+  println("Hello World")
+  var hW = "Hello World"
```

Comment:
hW is not getting value reassigned should be a val instead, also the name isn't great